<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>11</y>
      <width>411</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="key">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>130</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="value">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>200</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addBtn">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteBtn">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>280</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="findBtn">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>340</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>find</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="valueEdit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>340</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="printOrder">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>340</y>
      <width>201</width>
      <height>85</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="directBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>direct</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reverseBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>reverse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="IncreaseKeyBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>increaseKey</string>
    </property>
   </widget>
   <widget class="QPushButton" name="countLeftNodesBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>440</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>coutn</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
